package.json

```diff
@@ -218,10 +223,12 @@
 		"test-unit-php-multisite": "cross-env WP_MULTISITE=1 wp-scripts env test-php",
 		"test-unit:native": "cd test/native/ && cross-env NODE_ENV=test jest --config ./jest.config.js",
 		"test-unit:native:debug": "cd test/native/ && node --inspect ../../node_modules/.bin/jest --runInBand --config ./jest.config.js",
-		"playground:build": "npm run build:packages && parcel build playground/src/index.html -d playground/dist",
+		"playground:build": "npm run build:packages && parcel build playground/src/index.html -d playground/dist && npm run design-system:build",
 		"playground:start": "concurrently \"npm run dev:packages\" \"parcel playground/src/index.html -d playground/dist\"",
 		"preenv": "npm run check-engines",
-		"env": "wp-scripts env"
+		"env": "wp-scripts env",
+		"design-system:dev": "concurrently \"npm run dev:packages\" \"start-storybook -c ./packages/components/storybook\"",
```

Comment:
for playground we use `:start` and for gutenberg we use `:dev` I personally prefer start but we should try to consolidate all these similar commands on `:dev` maybe to avoid impacting the plugin devs.

Reply:
I've seen `start` used more to start production apps like web servers, so I kind of liked we opted for `dev`.

Reply:
Can we update the playground one?

Reply:
Done